client/src/components/Cart/PriceTotalWrapper/PriceTotalWrapper.tsx

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+import styled, { css } from 'styled-components';
+import { toKoreanMoneyFormatPure } from '../../../utils/moneyFormater';
+import { observer } from 'mobx-react-lite';
+import cartStore from '../../../stores/cartStore';
+
+import EQUAL from '../../../assets/images/equal.png';
+import PLUS from '../../../assets/images/plus.png';
+
+const PriceTotalWrapper = styled.div`
+  display: flex;
+  justify-content: flex-end;
+  align-items: center;
+  padding: 30px 40px 30px 40px;
+  margin-top: 30px;
+  border: 2px solid ${(props) => props.theme.color.grey2};
+`;
+
+const Wrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: flex-end;
+`;
+
+const Text = styled.div`
+  font-size: ${(props) => props.theme.fontSize.normal};
+`;
+
+const TextCount = styled.span`
+  font-weight: 600;
+`;
+
+const PriceWrapper = styled.div`
+  margin-top: 7px;
+`;
+
+type PriceProps = {
+  isTotal: boolean;
+};
+const Price = styled.span<PriceProps>`
+  font-size: ${(props) => props.theme.fontSize.medium};
+  font-weight: 700;
+
+  ${(props) =>
+    props.isTotal &&
+    css`
+      color: ${props.theme.color.mint2};
+    `};
+`;
+
+const Img = styled.img`
+  width: 22px;
+  height: 22px;
+  margin: 0 25px;
+`;
+
+const PriceTotal = (): JSX.Element => {
+  const cartItemList = cartStore.getCartItemList();
+
+  let totalPrice = 0;
+  for (const item of cartItemList) {
+    if (item.isSelected) {
+      totalPrice += item.price * item.count;
+    }
+  }
```

Comment:
filter, reduce와 같은 고차함수로 작성할 수도 있겠네요 ! :)
제 개인적인 취양인데 저는 filter, reduce가 좀 더 직관적이라고 생각이 드네요.. ㅎ

```suggestion
   const totalPrice = cartItemList
     .filter((cartItem) => cartItem.isSelected)
     .reduce((total, cartItem) => {
      total += cartItem.price * cartItem.count;
      return total;
    }, 0);
```

Reply:
막상 해놓고 보니까 비슷하네요 ..? 그냥 넘어가셔도 될 것 같습니다 .. ㅎ